setup-k3d/action.yaml

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Setup k3d'
+description: |
+  This action sets up a k3d cluster based on the supplied
+  configuration knobs.
+
+inputs:
+  k3s-version:
+    description: |
+      The version of k3s to use in the form: 1.27.x
+    required: false
+    default: 1.27.x
```

Comment:
I'd replace this with the image tag to use and default to `cgr.dev/chainguard/k3s:latest`